addon/templates/components/cfb-navigation.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each (reverse (take maxItems (reverse navigation.entries))) as |entry i|}}
```

Comment:
Haha, this looks awesome! :wink:

Reply:
"Awesome" - nice word for WEIRD :wink: This just takes the last n (`maxItems`) entries of the navigation. `ember-composable-helpers` didn't have a `take` for the last items so I just reversed it twice :man_shrugging:

Reply:
Totally makes sense, I like it!